docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,182 @@
+
+For example, suppose a user in the root cluster has the following role:
+
+```yaml
+spec:
+  allow:
+    logins: ["ubuntu"]
+```
+
+The role on the leaf cluster can be set up to use the user's allowed server
+accounts and names:
+
+```yaml
+spec:
+  allow:
+    logins: ["{{internal.logins}}"]
+```
+
+## Server role options
+
+The `allow` and `deny` sections described above are used for controlling
+the servers and logins allowed. Role options provide the Teleport features
+available for users with the specified role. These options apply to Server access.
+
+```yaml
+spec:
+  allow:
+    #....
+  options:
+    # Controls whether this role supports auto provisioning of users.
+    create_host_user: true
+    # forward_agent controls whether SSH agent forwarding is allowed
+    forward_agent: true
+    # port_forwarding controls whether TCP port forwarding is allowed for SSH
+    port_forwarding: true
+    # ssh_file_copy controls whether file copying (SCP/SFTP) is allowed.
+    # Defaults to true.
+    ssh_file_copy: false
+    # client_idle_timeout determines if SSH sessions to cluster nodes are
+    # forcefully terminated after no activity from a client (idle client).
+    # it overrides the global cluster setting. examples: "30m", "1h" or "1h30m"
+    client_idle_timeout: never
+    # Determines if the clients will be forcefully disconnected when their
+    # certificates expire in the middle of an active session.
+    # It overrides the global cluster setting.
+    disconnect_expired_cert: no
+    # max_sessions is total number of session channels that can be established
+    # across a single connection. Setting it to 10 matches OpenSSH default behavior.
+    max_sessions: 10
+    # Defines which events are recorded by the BPF-based session recorder.
+    enhanced_recording:
+    - command
+    - disk
+    - network
+    # permit_x11_forwarding allows users to use X11 forwarding with openssh
+    # clients and servers through the proxy
+    permit_x11_forwarding: true
+    # enterprise-only max_connections field sets a limit of concurrent sessions within a
+    # cluster. This setting slows down Teleport performance because it has to track
+    # connections cluster-wide.
+    max_connections: 2
+    # Define how Teleport deals with session recording failures, such as a full
+    # disk error. The value can be set to either `best_effort` or `strict`. If
+    # set to `strict`, the session will terminate immediately. If set to
+    # `best_effort`, the session won’t be terminated, and the recording will be
+    # disabled. The configuration is done per service (currently, only `ssh` is
+    # supported).
+    record_session:
+      # Optional: the default session recording mode to use when a
+      # protocol-specific mode is not set.
+      default: best_effort|strict
+      # Optional: Session recording mode for SSH sessions (Teleport Server
+      # Access). If not set, the value set on default will be used.
```

Comment:
```suggestion
      # Optional: Session recording mode for SSH sessions. 
      # If not set, the value set on default will be used.
```
Since "Server Access" is no longer a product